zgw/verzoek/src/main/kotlin/com/ritense/verzoek/VerzoekPluginEventListener.kt

```diff
@@ -91,6 +95,28 @@ open class VerzoekPluginEventListener(
                 "processDefinitionKey" to verzoekTypeProperties.processDefinitionKey,
                 "documentUrls" to getDocumentUrls(verzoekObjectData)
             )
+
+            verzoekProperties
+                .filter { it.copyStrategy == CopyStrategy.SPECIFIED }
+                .forEach { property ->
+                    property.mapping?.forEach {
+                        if (it.target.startsWith(PV_PREFIX)) {
+                            try {
+                                val key = it.target.substringAfter(delimiter = "/")
+                                val pointer = "/data/${it.source.substringAfter(delimiter = "/")}"
+
+                                val value = getJsonValueWithPointer(verzoekObjectData, pointer)
+                                verzoekVariables[key] = value
+
+                            } catch (e: Exception) {
```

Comment:
This is kind of a catch all. Is there any information that should be relayed to improve debugging here?